src/routes/github/setup/github-setup-get.ts

```diff
@@ -12,17 +17,40 @@ import { Installation } from "models/installation";
 		- If we have the users Jira host, redirect to marketplace.
 		- Otherwise, render the setup page (POST).
 */
+
+const getInstallationData = async (githubAppClient: GitHubAppClient | GitHubAPI, githubInstallationId: number, logger: Logger) => {
```

Comment:
Split the installation get to a seperate function for calrity and was able to handle the error cases more appropriately.